### ISSUE TITLE
Adjustable video offset/delay to compensate for audio delay in Foobar DSPs

### ISSUE THREAD

**Issue by amirzaidi** [sammoth/foo_mpv#1]
The player works great with zero-latency DSP plugins, but some DSPs like Ozone 8 (VST) offset the audio stream because they use look-ahead for some algorithms. This causes the video to be ahead of the audio track. Currently I'm disabling the DSPs causing this problem, but it would be nice if there was an adjustable audio sync option in milliseconds to compensate for this problem. In Ozone's case that would be around 400ms of a video delay.

**Comment by sammoth** [comment]
Yes that seems useful. Unfortunately none of mpv's settings can do this. Would it be useful to set it per-DSP preset or maybe even per-DSP somehow? I guess per-preset makes most sense as the delay will depend on Ozone's settings.

**Comment by amirzaidi** [comment]
I personally would be fine with either, as I don't ever change my preset. It's set up to do some basic post-processing for my headphones so it's always enabled. For others, yes, it might be great to have it linked to the current DSP preset somehow. I'm not familiar with Foobar's SDK so I don't know how much effort that would take.
